packages/common-scripts/index.d.ts

```diff
@@ -0,0 +1,85 @@
+import { TransactionSkeleton } from "@ckb-lumos/types"
+
+// TODO: secp256k1Blake160 types
+
+export declare const secp256k1Blake160Multisig: {
+  /**
+   * transfer capacity from multisig script cells
+   *
+   * @param txSkeleton
+   * @param fromAddress must be multisig script address
+   * @param fromInfo multisig parameters
+   * @param toAddress can be any type of lock script and can left empty
+   * @param amount transfer CKB capacity in shannon
+   * @param options
+   */
+  transfer(
+    txSkeleton: TransactionSkeleton,
+    fromAddress: string,
+    fromInfo: {
```

Comment:
I think it is better to define a separate type here for fromInfo, we can also explain what R and M does in the type

Reply:
Add an interface `MultisigScript`